pkg/install/generate.go

```diff
@@ -83,6 +81,12 @@ func InstallerPodSpec(
 				EmptyDir: &corev1.EmptyDirVolumeSource{},
 			},
 		},
+		{
+			Name: "logs",
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{},
+			},
+		},
```

Comment:
@dgoodwin I believe it's a better approach because there are limits on the size of the container FS, which limits usually fairly small (at least they are in OSD). I prefer the EmptyDir approach, but I'll remove it if you'd like.

@abhinavdahiya My understanding is that emptydir is not a ramdisk. They're scratch volumes.